YakimS/assignment4/IslandsCounterTest.java

```diff
@@ -1,44 +1,44 @@
 package com.shaya.CodeUAss4Package;
 
-import org.junit.Assert;
+import org.junit.Test;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
 
 public class IslandsCounterTest {
 
-    @org.junit.Test
+    @Test
     public void countIslandInExampleGrid()
     {

```

Comment:
Nit: this bracket should be in the same line as method name. Same for other methods below.